### PR TITLE
Update Android manual configuration pages

### PR DESCRIPTION
**Pull request by stefanosiano**
…tting started page

updated manual init page to contain manual sdk dependency

## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

removed manual-configuration.mdx as it's almost a duplicate of the getting started page
updated manual init page to contain manual sdk dependency

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Review by shanamatthews — APPROVED** [review]

File: src/platforms/android/configuration/manual-init.mdx [hunk 1]
```diff
@@ -1,11 +1,63 @@
 ---
-title: Manual Initialization
+title: Manual Configuration
 sidebar_order: 20
```
[the remark this answers is not in the document]
Comment (stefanosiano) [reply src/platforms/android/configuration/manual-init.mdx:3]: it works, thank for mentioning it!

File: src/platforms/android/configuration/manual-init.mdx [hunk 2]
```diff
@@ -1,11 +1,65 @@
...
+
+// Enable Java 1.8 source compatibility if you haven't yet.
+android {
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+}
+
+// Add Sentry's SDK as a dependency.
```
Comment (romtsn) [inline src/platforms/android/configuration/manual-init.mdx:47]:
```suggestion
```

File: src/platforms/android/configuration/manual-init.mdx [hunk 3]
```diff
@@ -1,11 +1,65 @@
...
+
+<Note>
+
+The [NDK integration](/platforms/android/using-ndk/) comes packaged with the SDK and requires API level 16, though other levels are supported. If you're using multiple Sentry dependencies, you can add a [bill of materials](/platforms/android/configuration/bill-of-materials) to avoid specifying the version of each dependency.
+
+</Note>
+
+## Manual Initialization
+
+If you don't want the Sentry Gradle plugin to install the Sentry SDK automatically, you can define the dependency directly in the `build.gradle` file of your app module.
```
Comment (romtsn) [inline src/platforms/android/configuration/manual-init.mdx:61]: hm, shouldn't this paragraph be under `## Manual Installation` actually? 🤔

Comment (stefanosiano) [reply src/platforms/android/configuration/manual-init.mdx:61]: yeah, moved the wrong sentence 😓

**Review by romtsn — APPROVED** [review]

**Pull request merged**